samples/dashboards/components/highcharts-column-assignment-keys-data/demo.js

```diff
@@ -0,0 +1,67 @@
+Dashboards.board('container', {
```

Comment:
When you edit the table you get unpredicted effects (points/labels are not removed)
![image](https://github.com/highcharts/highcharts/assets/57004120/7b74a244-e6b2-483e-8134-e475c13a0071)
Could you please investigate?

Reply:
It's a bug in Highcharts: https://jsfiddle.net/BlackLabel/30fy6roL/
Probably connected to #20687